### ISSUE TITLE
Set up continuous archiving and WAL archiving in Postgres, possibly using Barman?

### ISSUE THREAD

**Issue by rhliang** [cfe-lab/Kive#610]
As we get geared up to upgrade production Kive to v0.8, the procedure of backing up the database looms larger and larger.  Our current method, using `manage.py` to dump the database as JSON, gets slower and slower as our database grows.  The other time-consuming step -- backing up Datasets, Logs, and the like -- can be done while the system is still running, enabling us to run a much faster incremental backup during the upgrade downtime.  As such, the database backup is now the most time-consuming step.

I suggest we set up our database to perform continuous backups to the `/media/kive_backup` drive.  This can be done in real time without taking the database offline.  There's also a utility called Barman which apparently manages the backups for us, making it easier.

**Comment by rhliang** [comment]
I just set up Barman on Octomore, and it was way easier than I could have hoped.  When we take Bulbasaur offline next week for upgrade, I'd like to get this set up on Bulbasaur as well.  Note that this isn't even really anything to do with installing or running Kive, so I think it can and should be done in parallel to the deployment.

When I do that, I'll also set up some cron jobs for doing proper periodic backups of the data files.

**Comment by rhliang** [comment]
Just realized I could set up the data file backup cron jobs immediately, so I did that.  All the data folders will now be backed up daily at midnight by the `kivefleet` user.

**Comment by rhliang** [comment]
Turns out that backing up the sandboxes takes up too much space.  This wasn't a problem before, but now one of the pipelines used regularly by the lab creates huge sandboxes.  This is no problem, we can simply stop backing up these sandboxes.  They were never going to be reused by the system anyway.

Installing Barman on Bulbasaur hit several snags.  First, despite following the instructions on their website, I couldn't install it using `yum`.  I suppose the combination of CentOS 5 and PostgreSQL 8.4 is too ancient for their docs to be considering.  Then, I tried to install it from source, and that worked, but it won't run because the Python 2.7 interpreter we're using on Bulbasaur doesn't have the `bz2` module installed.  It must have been compiled without support, which makes sense because the `bzip2-devel` package wasn't installed until now.

For the moment, I've set up WAL archiving in Postgres, and we can try doing the database backups manually as per the instructions [here](https://www.postgresql.org/docs/8.4/static/backup-file.html) and [here](https://www.postgresql.org/docs/8.4/static/continuous-archiving.html).  At some point in the future, I might try installing a Python interpreter (maybe a Python 3 interpreter) local to either my own account or the `kivefleet` account, and setting up a virtualenv in which Barman can work.
